corehq/apps/accounting/models.py

```diff
@@ -1714,11 +1715,20 @@ def is_email_throttled(self):
 
     def email_context(self):
         context = super(BillingRecord, self).email_context()
+        total_balance = sum(invoice.balance for invoice in Invoice.objects.filter(
+            is_hidden=False,
+            subscription__subscriber__domain=self.invoice.get_domain(),
```

Comment:
it's django query syntax: https://docs.djangoproject.com/en/1.8/topics/db/queries/#field-lookups